site/ba.njk

```diff
@@ -0,0 +1,151 @@
+---
+layout: default
+meta_title: Pokret za zaustavljanje COVID-19 pandemije
+meta_description: Pridonesi borbi i spasi živote ne radeći nista! Jednostavno ostani jebeno kod kuće!
+body_class: home
+title: '#StayTheFuckHome'
+lang: en
+---
+
+<main id="main" class="main inner flex flex-1 flex-col py-10 lg:py-20 focus:outline-none" tabindex="-1">
+
+    <section id="container-centre" class="column centre flex-1">
+
+        <h1 class="page-title">{{ title }}</h1>
+        <h2 class="page-subtitle">Pokret za zaustavljanje COVID-19 pandemije</h2>
+
+        <div class="content">
+
+            <p class="text-xl">
+                Odgovorne agencije i medicinsko osoblje daju sve od sebe kako bi stavili pod kontrolu širenje 
+                <a href="https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2">SARS-CoV-2</a>
+                 virusa i ograničili <a href="https://en.wikipedia.org/wiki/Coronavirus_disease_2019">COVID-19 pandemiju.</a>
+				Nije vrijeme za sebičnost, i dužnost je svakog građanina na cijelom planetu da pridonese ovoj borbi.
+                <br /><br />
+                Ovog puta stvarno ne treba napraviti puno da bi potencijalno spasio nečiji zivot, i reći ću ovo jednostavno tako da svatko može razumjeti: <strong>Ostani jebeno kod kuće!</strong>
+            </p>
+
+            <h2 class="text-3xl">Manifesto kućne karantene (samoizolacije)</h2>
+
+            <p>
+                Bez dobro testiranih tretmana liječenja i vjerojatnosti da dobijemo djelotvorno cjepivo jos bar godinu dana, jedini djelotvoran način 
+                da se ova pandemija obuzda je dati ovom virusu manje šanse za širenje. Sljedeća lista je najučinkovitiji način za borbu protiv pandemije, i ove jednostavne stvari stvarno mogu pomoći da promijenimo stvari.
```

Comment:
i'd say "jednostavne mjere" and not "jednostavne stvari" so as not to have "stvari" two times in a sentence